draft-ietf-tls-dtls13.md

```diff
@@ -204,13 +204,13 @@ TLS cannot be used directly in datagram environments for the following five reas
    DTLS adds an acknowledgment message to enable better loss recovery.
 
 4. Handshake messages are potentially larger than can be contained in a single
-   datagram.  DTLS adds fields to handshake messages to support fragmentation
+   datagram.  DTLS added fields to handshake messages to support fragmentation
```

Comment:
```suggestion
   datagram.  DTLS adds fields to handshake messages to support fragmentation
```
Easier to stick with this to avoid people thinking about history